### PR TITLE
tab information

### PR DESCRIPTION
**Pull request by kpt96**
code Detail Screen (tab information)
Link redmine: https://edu-redmine.framgia.vn/issues/1231

### CODE REVIEW

**Comment by kpt96** [comment]
ready

File: app/src/main/res/layout/fragment_information.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+        <android.support.constraint.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <TextView
+                android:id="@+id/text_title_overview"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_information.xml]: text_title_over_view

File: app/src/main/res/layout/fragment_information.xml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                style="@style/StyleTextTitle"
+                android:layout_marginTop="?attr/actionBarSize"
+                android:text="@string/text_overview"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_information.xml]: text_over_view

File: app/src/main/res/layout/fragment_information.xml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/text_title_company"
+                app:recyclerAdapter="@{viewModel.companyAdapter}" />
+
+            <TextView
+                android:id="@+id/text_title_release_date"
+                style="@style/StyleTextTitle"
+                android:layout_marginTop="8dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_information.xml]: hardcode

**Comment by kpt96** [comment]
ready

**Pull request merged**